selftests/unit/test_utils_cpu.py

```diff
@@ -565,7 +564,7 @@ def test_set_cpuidle_state_withstateno(self):
     @unittest.skipUnless(recent_mock(),
                          "mock library version cannot (easily) patch open()")
     def test_set_cpuidle_state_withsetstate(self):
-        output = StringIO.StringIO()
+        output = io.BytesIO()
```

Comment:
yep, this should work on both, py2/py3 and the values will hopefully always be in ascii range (if not users should use other functions or encode manually)